projects/aca-playwright-shared/src/api/sites-api.ts

```diff
@@ -109,4 +109,38 @@ export class SitesApi {
       }
     }
   }
+
+  async createSiteMembershipRequestForPerson(personId: string, siteId: string): Promise<SiteMembershipRequestEntry> {
+    const body = {
+      id: siteId
+    };
+
+    try {
+      return await this.apiService.sites.createSiteMembershipRequestForPerson(personId, body);
+    } catch (error) {
+      console.error(`SitesApi createSiteMembershipRequestForPerson : catch : `, error);
+      return null;
+    }
+  }
+
+  async approveSiteMembershipRequest(siteId: string, inviteeId: string, opts?: {
```

Comment:
For now you don't need opts parameter as this function always is executed with 2 parameters. Opts parameter can be removed for now

Reply:
opts is removed in [aa865dc](https://github.com/Alfresco/alfresco-content-app/pull/3545/commits/aa865dcf243348da373c4560117ab99260fb3fe8)